src/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfiguration.tsx

```diff
@@ -153,11 +153,6 @@ const NetworkConfiguration = ({
       if (isMultiNodeCluster) {
         setFieldValue('machineNetworks', [], false);
       }
-    } else {
```

Comment:
It must be this changing that's causing problems when selecting DHCP having had empty apiVIp + ingressVIP.
I started with this state:
![initial-state](https://user-images.githubusercontent.com/829045/204481459-dc4333b0-855b-4873-a885-5b1b093b5d4e.png)


The flow I did was:
- User managed network --> Cluster managed network (OK)
- Then change from OVN to SDN (OK)
- Then enable DHCP, I'm getting what you see in the screenshot.


![api-error](https://user-images.githubusercontent.com/829045/204481546-2fad76c1-3cf5-4f40-945a-89e520652d11.png)